### ISSUE TITLE
Control.dispose() is never called

### ISSUE THREAD

**Issue by andreas-mausch** [msink/kotlin-libui#10]
Hi, me again. :)

I've noticed the dispose() method is never called for me in the examples.
I've tried hello-ktx and controlgallery, OS is Manjaro (4.19 Kernel).

The callback from uiOnShouldQuit is never reached.

Even when I dispose the mainWindow manually, I would have expected all children's dispose() method to be called - which was not the case!

I'm not sure if I missed something, or maybe it's a bug?

Andreas

**Comment by msink** [comment]
Yes, `dispose()` is not called when container class disposes children - it calls `free()`.
More precisely - libiu calls `ctl.pointed.Destroy`, and it calls `free()`.

About `uiOnShouldQuit()` not called - I don't know, maybe bug somewhere.
